dash-pipeline/SAI/specs/dash_eni.yaml

```diff
@@ -1338,8 +1338,8 @@ sai_apis:
     object_name: null
     allow_null: false
     valid_only: null
-    deprecated: false
     is_vlan: false
+    deprecated: false
```

Comment:
this change is odd, maybe it is coming from code merge, do you mind to help revert these changes?